luigi/static/visualiser/index.html

```diff
@@ -21,11 +21,10 @@
         <script src="lib/jquery.slimscroll.min.js"></script>
         <script src="lib/AdminLTE/js/app.min.js"></script>
         <script src="lib/datatables/js/jquery.dataTables.min.js"></script>
-        <script src="lib/URI.js/1.18.2/URI.js"></script>
+        <script src="lib/URI.js"></script>
```

Comment:
Wasn't the issue the `URI.js` matching some regex? Do we not have the same issue here?